### PR TITLE
Dev 25433

### PR DESCRIPTION
**Pull request by loaiabdelhalim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "@types/codemirror": "0.0.106",
     "@types/diff-match-patch": "^1.0.32",
     "@types/tinymce": "^4.6.0",
+    "codemirror6": "npm:codemirror@^6.0.0",
```
Comment (abhijeetnarvekar) [inline package.json:28]: Could you move this to dev dependency as its only needed for testing

**Review by abhijeetnarvekar — CHANGES_REQUESTED** [review]

**Comment by abhijeetnarvekar** [comment]
@loaiabdelhalim Pipeline isn't running, you might need to resolve git conflicts

File: package.json (hunk 1 above)
Comment (loaiabdelhalim) [reply package.json]: Done

**Comment by loaiabdelhalim** [comment]
> @loaiabdelhalim Pipeline isn't running, you might need to resolve git conflicts

Done

**Review by abhijeetnarvekar — APPROVED** [review]

**Review by JustinHorn — APPROVED** [review]

**Pull request merged**